### ISSUE TITLE
Error: Not enough space to copy the file (HENPLUS.BIN) to destination path (/dev_blind/hen/).

### ISSUE THREAD

**Issue by WolvenArch** [PS3-4K-Pro/PS3-4K-Pro#12]
Hi
I can't install or uninstall PS3 4K Pro

![image](https://github.com/PS3-4K-Pro/PS3-4K-Pro/assets/165651440/0d749f2d-d39b-428c-97f4-5d3227d93620)

There is no HEN enabler icon and the Internet browser is not working (The page cannot be displayed. (80710102).

I went from a fully functional PS3 after a week of installing games and copying roms; now it's fully unfunctional.

Luckily, I was able to reinstall HFW.

I hope it helps in some way.

**Comment by LuanTeles** [comment]
Hey, sorry to hear that you had issues. This problem usually occurs when the flash partition is full. Were you using the official HEN, or have you installed any XMB mods?Regarding the second issue, it seems to be a DNS problem. It could be related to the PS3Xploit site or the DNS configuration of your PS3/router. Setting up Google DNS settings might fix it.And yes, you can always reinstall the firmware to resolve any software-related issues.

**Comment by WolvenArch** [comment]
Oh hey, nothing to be sorry about; it's all part of the take own risk after all.
My goal is to share the experience, so that will probably help.
I am not sure what XMB mods are; I only have the usual tools: webmanMOD, Mutliman, Irisman, ManaGunZ, PSN Patch, PS3 Unlock HDD Space, and prepISO.

One thing I did not mention is that I installed it first using that Arisen Studio tool, which did just copy the package to packages folder of hdd0, then I installed it from XMB HEN's Package Manager, However, it was version 5.1:
![image](https://github.com/user-attachments/assets/afc461b9-e975-406a-941a-72cfc623e9e4)

I also installed before it the Hen Toolbox:
![image](https://github.com/user-attachments/assets/1180b3f8-2b1d-45d1-acb4-75e18c6d551f)

Even that after all what happened I understood It was a mistake to install Hex Toolbox since it's outdated.

then I went and downloaded the latest package in the releases here in git, and this is where it was not able to install and got me that error in the first picture.

I hope it helps in finding a fix, and probably a new version will be able to make users happy about it, after all, even when I did install it, a lot of GUI was glitching, some big stretched white PS icon overlays, and when enabling HEN, the PS3 4K Pro folders disappeared.

That's all folks, and thank you very much, @LuanTeles for your time I really appreciate that.
